Source/Scene/ImplicitTileCoordinates.js

```diff
@@ -178,12 +219,322 @@ ImplicitTileCoordinates.prototype.deriveChildCoordinates = function (
+  });
+};
+
+/**
+ * Get the local coordinates of the child subtree relative to its subtree
+ *
+ * @returns {ImplicitTileCoordinates} The local coordinates of the child subtree
+ * @private
+ */
+ImplicitTileCoordinates.prototype.deriveLocalChildSubtreeCoordinates = function () {
+  //>>includeStart('debug', pragmas.debug);
+  if (!this.isRootOfSubtree() || this.isRoot()) {
+    throw new DeveloperError(
+      "child subtree must be the root of a non-root subtree"
+    );
+  }
+  //>>includeEnd('debug');
+
+  var subdivisionScheme = this.subdivisionScheme;
+  var subtreeLevels = this.subtreeLevels;
+  var tileX = this.x;
+  var tileY = this.y;
+
+  var localX = tileX % (1 << subtreeLevels);
+  var localY = tileY % (1 << subtreeLevels);
+
+  if (subdivisionScheme === ImplicitSubdivisionScheme.OCTREE) {
+    var tileZ = this.z;
+    var localZ = tileZ % (1 << subtreeLevels);
+
+    return new ImplicitTileCoordinates({
+      subdivisionScheme: subdivisionScheme,
+      subtreeLevels: subtreeLevels,
+      level: subtreeLevels,
+      x: localX,
+      y: localY,
+      z: localZ,
+    });
+  }
+
+  // Quadtree
+  return new ImplicitTileCoordinates({
+    subdivisionScheme: subdivisionScheme,
+    subtreeLevels: subtreeLevels,
+    level: subtreeLevels,
+    x: localX,
+    y: localY,
+  });
+};
+
+/**
+ * Returns whether this tile is an ancestor of another tile
+ *
+ * @param {ImplicitTileCoordinates} otherCoordinates the other tile coordinates
+ * @returns {Boolean} <code>true</code> if this tile is an ancestor of the other tile
+ * @private
+ */
+ImplicitTileCoordinates.prototype.isAncestorOf = function (otherCoordinates) {
+  var subdivisionScheme = this.subdivisionScheme;
+  var subtreeLevels = this.subtreeLevels;
+  var tileLevel = this.level;
+  var tileX = this.x;
+  var tileY = this.y;
+
+  //>>includeStart('debug', pragmas.debug);
+  Check.typeOf.object("otherCoordinates", otherCoordinates);
+  if (subdivisionScheme !== otherCoordinates.subdivisionScheme) {
+    throw new DeveloperError("coordinates must have same subdivisionScheme");
+  }
+
+  if (subtreeLevels !== otherCoordinates.subtreeLevels) {
+    throw new DeveloperError("coordinates must have same subtreeLevels");
+  }
+  //>>includeEnd('debug');
+
+  var levelDifference = otherCoordinates.level - tileLevel;
+  if (levelDifference <= 0) {
+    return false;
+  }
+
+  var isAncestorX = tileX === otherCoordinates.x >> levelDifference;
+  var isAncestorY = tileY === otherCoordinates.y >> levelDifference;
+
+  if (subdivisionScheme === ImplicitSubdivisionScheme.OCTREE) {
+    var tileZ = this.z;
+    var isAncestorZ = tileZ === otherCoordinates.z >> levelDifference;
+
+    return isAncestorX && isAncestorY && isAncestorZ;
+  }
+
+  // Quadtree
+  return isAncestorX && isAncestorY;
+};
+
+/**
+ * Returns whether this tile is the root
+ *
+ * @returns {Boolean} <code>true</code> if this tile is the root
+ * @private
+ */
+ImplicitTileCoordinates.prototype.isRoot = function () {
```

Comment:
Is this too ambiguous? There's `isRootOfSubtree` but the difference could be easy to miss